modular_skyrat/master_files/code/datums/traits/neutral.dm

```diff
@@ -69,7 +69,7 @@ GLOBAL_VAR_INIT(DNR_trait_overlay, generate_DNR_trait_overlay())
 
 /// Adds the DNR HUD element if src has TRAIT_DNR. Removes it otherwise.
 /mob/living/proc/update_dnr_hud()
-/* 	var/image/dnr_holder = hud_list?[DNR_HUD] // BUBBER EDIT - more DNR fixing
```

Comment:
i really don't want to reintroduce skyrat code when the tg impl works better

Reply:
also given i've removed most sources that apply this trait and will continue to do so, because it's simply un-needed

Reply:
see comment above, this is not reimplementing TGcode, this is standalone functionality

Reply:
https://github.com/Skyrat-SS13/Skyrat-tg/pull/23524

Reply:
oh ok, i just am afraid of reintroducing the skyrat way of the skull icon showing up.